Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2511-add-columns-admin-panel

Resolves # EREGCSC-2511 Add Columns admin panel

**Description-**
Added following fields to uploaded files

- Date
- Document name
- Category
- Updated at
- Index populated 

All uploaded file fields are sortable except "Index populated"
Included index populated to supplemental content
Included index populated to FR docs.

**This pull request changes...**
file_manager/admin.py is modified to include the additional fields and configure sorting.
resources/admin.py is modifed to include "index populated"  in supplemental content and federal register doc
a migration file is include so that we can add updated_at
updated_at added to the model so that we can display it in "File manager" 
updated_at was also added to the serializer. 

**Steps to manually verify this change...**

steps to view and verify change
sign into the [admin panel](https://3w7oh6w9gd.execute-api.us-east-1.amazonaws.com/dev1188/admin/)
visit [Uploaded Files](https://3w7oh6w9gd.execute-api.us-east-1.amazonaws.com/dev1188/admin/file_manager/uploadedfile/)
 
![Screen Shot 2024-02-29 at 12 10 13 PM](https://github.com/Enterprise-CMCS/cmcs-eregulations/assets/7727136/63606fb9-3802-4876-9a90-88c997c6a564)
All fields are sortable except Index Populated.
Notice the following fields are displayed
- Date
- Document name
- Category
- Updated at
- Index populated 
Visit [Supplemental Content](https://3w7oh6w9gd.execute-api.us-east-1.amazonaws.com/dev1188/admin/resources/supplementalcontent/)
Notice "Index Poulated" is displayed
![Screen Shot 2024-02-29 at 12 12 42 PM](https://github.com/Enterprise-CMCS/cmcs-eregulations/assets/7727136/f016e80a-ac13-4c33-9f87-5383026ef607)
Visit [Federal register documents](https://3w7oh6w9gd.execute-api.us-east-1.amazonaws.com/dev1188/admin/resources/federalregisterdocument/)
Notice "Index Poulated" is displayed
![Screen Shot 2024-02-29 at 12 25 02 PM](https://github.com/Enterprise-CMCS/cmcs-eregulations/assets/7727136/eb205856-55ec-460f-bee9-0d452d3d8e66)
